core/arch/arm/kernel/entry_a64.S

```diff
@@ -382,14 +382,27 @@ clear_nex_bss:
 #endif
 	bl	boot_init_primary_late
 #ifdef CFG_CORE_PAUTH
-	init_pauth_per_cpu
```

Comment:
Since `init_pauth_per_cpu` is no longer applicable to the primary core, how about renaming it? `init_pauth_secondary_(core|cpu)` for instance.

Reply:
OK